core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/value/ValueSelectorFactory.java

```diff
@@ -256,71 +257,64 @@ protected ValueSelector<Solution_> applyInitializedChainedValueFilter(HeuristicC
     }
 
     protected void validateSorting(SelectionOrder resolvedSelectionOrder) {
-        if ((config.getSorterManner() != null || config.getSorterComparatorClass() != null
+        var sorterManner = config.getSorterManner();
```

Comment:
Consider applying the same pattern used in multiple places: declare the variable and then check if it is null.